src/lib.rs

```diff
@@ -396,14 +400,16 @@ impl GooseAttack {
     /// let mut goose_attack = GooseAttack::initialize();
     /// ```
     pub fn initialize() -> Result<GooseAttack, GooseError> {
+        let configuration = GooseConfiguration::parse_args_default_or_exit();
         Ok(GooseAttack {
             test_start_task: None,
             test_stop_task: None,
             task_sets: Vec::new(),
             weighted_users: Vec::new(),
             weighted_gaggle_users: Vec::new(),
             defaults: GooseDefaults::default(),
-            configuration: GooseConfiguration::parse_args_default_or_exit(),
+            graph_data: GraphData::new(configuration.no_reset_metrics),
```

Comment:
Why disable startup metrics in the charts? It seems we collect the startup statistics either way, so IMO we should always show them. Or is there some reason they're not reliable?

Reply:
Done.